features/issues/components/issue-filters/issue-filters.tsx

```diff
@@ -0,0 +1,102 @@
+import { useState, useRef, ChangeEvent } from "react";
+import { Select } from "@features/ui";
+import { Input, InputIcon } from "@features/ui";
+import { Button, ButtonSize, ButtonIcon } from "@features/ui";
+import { statusOptions, levelOptions } from "./options";
+import { useRouter } from "next/router";
+import { debounce } from "lodash";
+import styles from "./issue-filters.module.scss";
+
+export function IssueFilters() {
+  const router = useRouter();
+
+  const [filters, setFilters] = useState({
+    status: router.query.status || "",
+    level: router.query.level || "",
+    project: router.query.project || "",
+  });
+
+  const updateFilter = (filterName: string, filterValue: string) => {
```

Comment:
It would be better to accept an object as parameter here. In theory you can pass any filter name here even non-existent ones. You might introduce nasty bugs because of typos in the filter name.

I'd recommend accepting partial filter objects like

```
{
  status: '...'
}
```

You can type this like this

```
const updateFilter = (updatedFilter: Partial<IssueFilters>) => {
```